pup-0001.md

```diff
@@ -85,31 +87,29 @@ additional commits to the pull request either as patches or additional pull requ
 
 ### Voting
 
+The lazy consensus model is used for deciding via votes on the mailing list.
+
 The +1, +0, -0, -1 voting process is used; votes are sent as responses via the
-announcement/discussion mailing list. The voting has the following meanings:
+announcement/discussion thread on the mailing list. The voting has the following meanings:
```

Comment:
Could another sentence explicitly stating that a vote can be changed at any time in response to other people's concerns and community votes be added?